bootstrap

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+autoreconf -f -i
```

Comment:
If autoreconf fails, this script will return 0.
Could you add a "set -e" at its beginning, please?